### PR TITLE
Opentelemetry azure monitor release update

### PR DESCRIPTION
**Pull request by lzchen**

### CODE REVIEW

File: releases/2021-01/python.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ The Azure SDK team is pleased to make available the January 2021 client library
 #### Updates
 
 - Event Hubs
+- Opentelemetry Azure Monitor
```
Comment (xiangyan99) [inline releases/2021-01/python.md:18]: I think this should be beta not updates?

**Comment by lzchen** [comment]
@xiangyan99 
Any idea what this build error is referring to?

**Review by xiangyan99 — APPROVED** [review]

**Review by xiangyan99 — COMMENTED** [review]
Seems like we forget to add the installation script.

**Pull request merged**